LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Herman Starikov
+Copyright (c) 2020 Gago
```

Comment:
pretty sure this was unintentional, but https://law.stackexchange.com/questions/16847/how-should-multiple-copyright-holders-be-credited-using-the-mit-license

Reply:
oh, yea, this is cp, from what I had. reverting!